rust/Makefile

```diff
@@ -1,6 +1,7 @@
 .DEFAULT_GOAL := help
 PROJECTNAME=peaq_codec_api
-LLVM_PATH=/usr/local/homebrew/opt/llvm
+# LLVM_PATH=/usr/local/homebrew/opt/llvm
+LLVM_PATH=/opt/homebrew/opt/llvm
```

Comment:
Does that mean when we build this project, we have to install the LLVM into this path, `/opt/homebrew/opt/llvm`?

Reply:
LLVM is needed to build the android binaries;  getting the path from Env var is a better approach. I will do that now.

Reply:
Changed